Tests/xcprojectlint-packageTests/NoDanglingFilesTests.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import XCTest
+@testable import xcprojectlint_package
+
+class noDanglingFilesTests: XCTestCase {
+
+    func testEmptyGroupReturnsError() {
```

Comment:
I think I know where you copied this from. :-) Rename to match the contents of the test, please.